static/scss/answers/cards/product-prominentimage-clickable.scss

```diff
@@ -77,15 +77,20 @@
   &-imgWrapper
   {
     display: flex;
+    flex-direction: column;
     align-items: flex-start;
+    justify-content: center;
     width: 100%;
     flex-shrink: 0;
   }
 
   &-img
   {
     width: 100%;
-    height: auto;
+    height: 300px;
```

Comment:
just checking that there's no drawbacks to having a fixed height that Rose might care about?

Reply:
For uniform size, I think a fixed height have to be implemented. This was used in the HH link from the jira item as well. But it seems like Rose is reconsidering this change and leaving the images as what we have before.